lib/cc/cli/validate_config.rb

```diff
@@ -4,95 +4,63 @@ module CC
   module CLI
     class ValidateConfig < Command
       SHORT_HELP = "Validate your .codeclimate.yml.".freeze
-
-      include CC::Analyzer
-      include CC::Yaml
-
       VALID_CONFIG_MESSAGE = "No errors or warnings found in .codeclimate.yml file.".freeze
 
       def run
         require_codeclimate_yml
-        verify_yaml
-      end
-
-      private
+        process_args
 
-      def verify_yaml
         if any_issues?
           display_issues
         else
-          puts colorize(VALID_CONFIG_MESSAGE, :green)
+          puts VALID_CONFIG_MESSAGE
         end
-      end
 
-      def any_issues?
-        parsed_yaml.errors? || parsed_yaml.nested_warnings.any? || parsed_yaml.warnings? || invalid_engines.any?
+        exit 1 unless validator.valid?
```

Comment:
Oh, man, another case where we've just been always exiting 0 no matter what? Good catch to fix.